subsys/caf/modules/buttons.c

```diff
@@ -404,7 +404,9 @@ static void button_pressed_fn(struct k_work *work)
 
 	switch (state) {
 	case STATE_IDLE:
-		EVENT_SUBMIT(new_wake_up_event());
+		if (IS_ENABLED(CONFIG_CAF_BUTTONS_PM_EVENTS)) {
```

Comment:
`CONFIG_CAF_BUTTONS_PM_EVENTS` was intended to enable **reaction** on power events. Update the documentation if we want to keep this.

Reply:
Fixed